doc/python/lines-on-mapbox.md

```diff
@@ -94,15 +91,53 @@ for feature, name in zip(geo_df.geometry, geo_df.name):
         lons = np.append(lons, None)
         names = np.append(names, None)
 
-fig = px.line_mapbox(lat=lats, lon=lons, hover_name=names,
-                     mapbox_style="open-street-map", zoom=1)
+fig = px.line_map(lat=lats, lon=lons, hover_name=names,
```

Comment:
This example required installation of `wget` i.e. to download 
```py
wget.download("https://plotly.github.io/datasets/ne_50m_rivers_lake_centerlines.zip")
```
Wondering if we could simplify that or alternatively put `wget` and other required modules e.g. `datashader`  in requirements?

Reply:
they are in the docs requirements here https://github.com/plotly/plotly.py/blob/master/doc/requirements.txt and those are used when the docs are built